lib/socketwrench.js

```diff
@@ -192,7 +192,14 @@
 
         for (property in connectionData) {
           if (connectionData.hasOwnProperty(property)) {
-            parameters.push(property + '=' + connectionData[property]);
+            var propertyData = connectionData[property];
+            if ('[object Array]' == Object.prototype.toString.call(propertyData)) {
```

Comment:
Please use [*Threequals*](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators#Identity_strict_equality_()) for comparison when the operands are of the same *type*.